static_src/util/uaa_api.js

```diff
@@ -12,5 +12,27 @@ export default {
   fetchUaaInfo(guid) {
     return http.get(`${URL}/uaainfo?uaa_guid=${guid}`)
       .then(res => res.data);
+  },
+
+  getInviteUaaUser(email) {
```

Comment:
If this is a post request it should be specified as such in the function name. Maybe `postInviteUaaUser` or even use `inviteUaaUser`.